src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -710,7 +710,7 @@ public function getRelation($name)
      * @param  string  $relation
      * @return array
      */
-    public function nestedRelations($relation)
+    protected function nestedRelations($relation)
```

Comment:
This method has been public for a few weeks now but I think it's better to change it back to protected since it's not a part of the public API.